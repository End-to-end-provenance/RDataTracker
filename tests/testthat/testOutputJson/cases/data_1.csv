"ddg.type","ddg.num","ddg.name","ddg.value","ddg.val.type","ddg.scope","ddg.from.env","ddg.time","ddg.hash","ddg.rw","ddg.loc"
"Data",1,"f","#ddg.function","{""container"":""vector"", ""dimension"":[1], ""type"":[""character""]}","R_GlobalEnv",FALSE,"","","",""
