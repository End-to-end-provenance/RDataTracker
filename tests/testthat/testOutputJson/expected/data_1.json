
		// data nodes
		"rdt:d1": {
			"rdt:name": "f",
			"rdt:value": "#ddg.function",
			"rdt:valType": "{\"container\":\"vector\", \"dimension\":[1], \"type\":[\"character\"]}",
			"rdt:type": "Data",
			"rdt:scope": "R_GlobalEnv",
			"rdt:fromEnv": false,
			"rdt:hash": "",
			"rdt:timestamp": "",
			"rdt:location": ""
		}
